src/browser/api_protocol/api_handlers/system.ts

```diff
@@ -14,8 +14,10 @@ import {
     Nacker,
     NackerError,
     NackerErrorString,
+    PreloadScript,
     StartManifest
 } from '../../../shapes';
+import { DownloadResult } from '../../../browser/preload_scripts';
```

Comment:
Would it be better to move `DownloadResult` to shapes since it's being used in more than one place?

Reply:
yeah, I had the same question to @whyn07m3 . He suggested to keep it in the same file now.